src/backend/joanie/payment/backends/payplug/__init__.py

```diff
@@ -188,7 +188,7 @@ def handle_notification(self, request):
         """
         try:
             resource = notifications.treat(request.body)
-        except UnknownAPIResource as error:
```

Comment:
Isn't there other cases that were caught by this UnknownAPIResource? In your commit message it is not clear what this exception was catching vs the PayplugError and where it came from.

Reply:
It cames from Payplug package. `UnknownAPIResource` inherits from `PayplugError`. To be honest, I do not remember why I decided to catch only `UnknownAPIResource` exception when I wrote this logic but during my tests last friday, I realized we could encount several Payplug error (i.e: The payment resource provided is not found on Payplug, Payplug service is down, etc.) That's why I decided to catch all kind of PayplugError.